exporter/datadogexporter/internal/metadata/metadata.go

```diff
@@ -150,6 +150,11 @@ func fillHostMetadata(params component.ExporterCreateSettings, cfg *config.Confi
 }
 
 func pushMetadata(cfg *config.Config, buildInfo component.BuildInfo, metadata *HostMetadata) error {
+	if metadata.Meta.Hostname == "" {
```

Comment:
Is this considered an error? Should it be logged somehow? I feel like just ignoring this data point will cause confusion.

Reply:
It's not an error (in serverless environments we don't have a hostname). I added a debug log, seems like a good idea.